recipes/libgd/all/conanfile.py

```diff
@@ -55,7 +55,8 @@ def requirements(self):
         self.requires("zlib/1.2.11")
         if self.options.with_png:
             self.requires("libpng/1.6.37")
-            self.requires("getopt-for-visual-studio/20200201")
+            if self.settings.compiler == "Visual Studio":
```

Comment:
```suggestion
            if is_msvc(self):
```
from conan.tools.microsoft import is_msvc

Reply:
Thanks for the suggestion! However, `is_msvc` was added in [1.45.0](https://github.com/conan-io/conan/releases/tag/1.45.0), and the current conanfile specifies the following: `required_conan_version = ">=1.33.0"`, so I will also update that line.